integreat_cms/static/src/js/media-management/component/breadcrumbs.tsx

```diff
@@ -48,6 +52,14 @@ const Breadcrumbs = ({ breadCrumbs, searchQuery, mediaTranslations, allowDrop, d
                         {mediaTranslations.heading_media_root}
                     </Link>
                 </li>
+                {isUnusedFiltered && (
+                    <>
+                        <li class="py-2">
+                            <ChevronRight />
+                        </li>
+                        <li class="px-3 py-2">{mediaTranslations.heading_show_unused_files}</li>
+                    </>
+                )}
```

Comment:
This returns weird results when in a subdirectory:
![Screenshot 2023-03-04 at 14-25-19 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/222904604-930449c2-616f-456d-929c-b402fbee1710.png)